packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -815,7 +823,39 @@ Iterable<File> findApkFiles(
       fileExtension: '.apk',
     );
   }
-  return apks;
+  return apks.map((File file) => file.path);
```

Comment:
This function could just become:

```dart
Iterable<String> findApkFilesModule(
  FlutterProject project,
  AndroidBuildInfo androidBuildInfo,
) {
   final Iterable<String> apkFileNames = _apkFilesFor(androidBuildInfo);
   final Directory apkDirectory = getApkDirectory(project);
   return apkFileNames.map((File file) => fs.path.join(apkDirectory.path, file.path));
}
```

Reply:
oh nvm. This is for `build apk` in a module, right?

Reply:
Yeah